### PR TITLE
Minor script fixes for train.py

### PR DESCRIPTION
**Pull request by jsjason**
- Added command line argument `num_iters`
- Added command line argument `log_period`
- Print current loss correctly when logging
  - e.g. `INFO:tensorflow:global step 250: loss = 4.9927 (0.961 sec/step)`

### CODE REVIEW

**Review by anyj0527 — APPROVED** [review]
lgtm

**Comment by anyj0527** [comment]
Hooray jason!
